python/kubeflow/trainer/api/trainer_client_test.py

```diff
@@ -695,6 +703,27 @@ def test_list_runtimes(training_client, test_case):
                 train_job_trainer=get_custom_trainer(),
             ),
         ),
+        TestCase(
+            name="valid flow with custom trainer and env vars",
+            expected_status=SUCCESS,
+            config={
+                "trainer": types.CustomTrainer(
+                    func=lambda: print("Hello World"),
+                    func_args={"learning_rate": 0.001, "batch_size": 32},
+                    packages_to_install=["torch", "numpy"],
+                    pip_index_url=constants.DEFAULT_PIP_INDEX_URL,
+                    num_nodes=2,
+                    env={
+                        "TEST_ENV": "test_value",
+                        "ANOTHER_ENV": "another_value",
+                    },
+                )
+            },
+            expected_output=get_train_job(
+                train_job_name=TRAIN_JOB_WITH_CUSTOM_TRAINER_ENV,
+                train_job_trainer=get_custom_trainer(include_env=True),
```

Comment:
Instead of passing boolean to `the get_custom_trainer()`, can we directly pass the env list, it looks clearer, I think.
```suggestion
                train_job_trainer=get_custom_trainer([
                    models.IoK8sApiCoreV1EnvVar(name="TEST_ENV", value="test_value"),
                    models.IoK8sApiCoreV1EnvVar(name="ANOTHER_ENV", value="another_value"),
                ]
                ),
```

Reply:
Yes I agree, I kept the old approach for consistency but I prefer this. Let me change it.